src/Gui/Action.cpp

```diff
@@ -1051,14 +1051,13 @@ void RecentMacrosAction::setFiles(const QStringList& files)
     if (!existingCommands.isEmpty()) {
         auto msgMain = QStringLiteral("Recent macros : keyboard shortcut(s)");
         for (int index = 0; index < accel_col.count(); index++) {
-            msgMain = msgMain + QStringLiteral(" %1").arg(accel_col[index]);
+            msgMain += QStringLiteral(" %1").arg(accel_col[index]);
         }
-        msgMain = msgMain + QStringLiteral(" disabled because of conflicts with");
+        msgMain += QStringLiteral(" disabled because of conflicts with");
         for (int index = 0; index < existingCommands.count(); index++) {
-            msgMain = msgMain + QStringLiteral(" %1").arg(existingCommands[index]);
+            msgMain += QStringLiteral(" %1").arg(existingCommands[index]);
         }
-        msgMain = msgMain
-            + QStringLiteral(" respectively.\nHint: In Preferences -> Python -> Macro ->"
+        msgMain += QStringLiteral(" respectively.\nHint: In Preferences -> Python -> Macro ->"

```

Comment:
```suggestion
        msgMain += 
            QStringLiteral(" respectively.\nHint: In Preferences -> Python -> Macro ->" 
```
clang-format puts it like this, personally your proposal does look better to me so I'm merging anyways. If we eventually run clang-format on these files it will be changed anyways.

Reply:
You beat me to it :smile:   
Thanks